library/cwm/src/lib/cwm/abstract_widget.rb

```diff
@@ -1,6 +1,22 @@
 require "abstract_method"
 
 module CWM
+  # A Yast::Term that can be passed as is to Yast::UI methods
+  # (OpenDialog, ReplaceWidget)
+  class UITerm < Yast::Term; end
```

Comment:
well, I am fine with it. Maybe mention in those three terms, that it is three steps of processing.
`WidgetTerm -> StringTerm -> UITerm` first processing expands CWM::AbstractWidget and create StringTerm, which is later processed to UITerm by old part of CWM.